src/card.css

```diff
@@ -0,0 +1,104 @@
+@import "./cardStyles/reset.css";
+@import "./cardStyles/normalize.css";
+@import "./cardStyles/variables.css";
+@import "./cardStyles/stars.css";
```

Comment:
You could move these imports to a separate file (e.g. `main.css`)

Reply:
Good sugestion. Code will become mor user firendly. Done